src/delimiter.js

```diff
@@ -329,14 +329,12 @@ const sqrtSvg = function(sqrtName, height, viewBoxHeight, options) {
     }
     const pathNode = new domTree.pathNode(sqrtName, alternate);
 
-    let attributes = [["width", "100%"], ["height", height + "em"]];
+    const attributes = [["width", "400em"], ["height", height + "em"]];
     attributes.push(["viewBox", "0 0 400000 " + viewBoxHeight]);
```

Comment:
It'd be good to compute the `width` attribute in terms of the `viewBox`'s width (or vice versa) so that they stay in sync.

Reply:
I've consistently used 400 ems for all the wide paths. I suppose that might not be obvious to someone looking at this cold. I'll look at it.

Reply:
The comment is good enough for now.

Reply:
> The comment is good enough for now.

Let's suppose that someone other than me comes along and writes a PR with an SVG in it. They will need to coordinate `viewBox` geometry and document geometry. They will need to realize this early, when they are arranging the `<path>` geometry.  So I am trying to leave multiple comments regarding the 1000:1 ratio. 

Such is my reasoning for writing `400em`. Reasoning with which you may disagree, but there it is.

Reply:
Would it make sense to define a `const hugeWidth = '400em'` somewhere and use that variable repeatedly, instead of repeating a constant like this?  (And one comment could explain why this choice is sensible.)  Sorry if this suggestion doesn't make sense, coming from out of context.

Reply:
A constant for 400em would do no harm. But the more critical concept is the 1000:1 ratio between ems and `viewBox` geometry. Some future contributor could waste a lot of time if they don't realize that early in their work. So any constant, defined in one place, would be improved by adding multiple comments, scattered widely.